### PR TITLE
cart view remove item 6

### PR DESCRIPTION
**Pull request by valerietrudell**
removes item link
shows each item's image
each items title is a link
when you remove a link there is a flash message and there is a link in the flash message to the item show page

### CODE REVIEW

**Comment by valerietrudell** [comment]
@meyerhoferc Hey Courtney, Charolotte is  looking at my pull request to merge my updated branch into master will you look at it also?

File: spec/features/cart/user_can_add_items_to_cart_spec.rb [hunk 1]
```diff
@@ -36,7 +39,30 @@
 
     within('.notice') do
       expect(page).to have_content('Love Potion added to cart')
+
     end
   end
 
 end
+
+describe "user views cart" do
```
Comment (CjMoore) [inline spec/features/cart/user_can_add_items_to_cart_spec.rb]: I would take this test out for now. Just because we're skipping it right now

**Comment by CjMoore** [comment]
Made a couple changes, but otherwise it looks great. Ready for merge after check #2 @meyerhoferc .

File: spec/features/cart/user_can_add_items_to_cart_spec.rb [hunk 2]
```diff
@@ -26,3 +26,21 @@
     end
   end
 end
+
+xdescribe "user views cart" do
+  it "user sees multiple items in cart" do
+    # item_1 =  Fabricate(:item)
```
Comment (meyerhoferc) [inline spec/features/cart/user_can_add_items_to_cart_spec.rb]: there's a lot of commented out code in this test, can we delete it?

File: spec/features/cart/user_can_add_items_to_cart_spec.rb [hunk 3]
```diff
@@ -20,27 +20,33 @@
...
+    item_two = category.items.create!(title:"Elder wand", description: "Derp", price: 20.22)
+
+    item_three = category.items.create!(title:"Elephant wand", description: "things", price: 13.22)
+
+    items = [item_one, item_two, item_three]
+    cart = Cart.new(items)
+    visit cart_path
+    save_and_open_page
+    #byebug
+    # table_cell = page.all("table tr")
```
Comment (meyerhoferc) [inline spec/features/cart/user_can_add_items_to_cart_spec.rb]: comments here too

File: spec/features/cart/user_can_add_items_to_cart_spec.rb [hunk 4]
```diff
@@ -20,27 +20,33 @@
...
+
+    # expect(table_cell[0]).to have_content(item_1.title)
+    expect(page).to have_content(item_one.title)
+    expect(page).to have_content(item_two.title)
+    expect(page).to have_content(item_three.title)
```
Comment (meyerhoferc) [inline spec/features/cart/user_can_add_items_to_cart_spec.rb]: can we add a CSS selector for these assertions?

File: spec/features/cart/user_can_add_items_to_cart_spec.rb [hunk 5]
```diff
@@ -29,7 +29,7 @@
 end
 
 describe "user views cart" do
-  it "user sees multiple items in cart" do
+  xit "user sees multiple items in cart" do
     category = Category.create!(name: "wands")
     item_one = category.items.create!(title:"Fox wand", description: "Does magical things", price: 15.22)
```
Comment (meyerhoferc) [inline spec/features/cart/user_can_add_items_to_cart_spec.rb]: it looks like this test is skipped. does this test pas?

**Pull request merged**